.aws/task-definition.json

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
I think the task definitions should be on a separate project. But you can double-check, just in case.